src/main/java/com/continuum/nova/Mouse.java

```diff
@@ -0,0 +1,248 @@
+package com.continuum.nova;
+//
+// Source code recreated from a .class file by IntelliJ IDEA
```

Comment:
Did you decompile the LWJGL mouse class? Why not just extend it and override its methods?